### PR TITLE
BASIQ-59: Absolute Path

### PR DESCRIPTION
**Pull request by ardacoskun**
Checklist for Pull Requests

- [x] Run 'yarn lint'
- [x] Remove console.log() statements if unnecessary
- [x] Remove all debugger statements
- [x] Code formatting

### CODE REVIEW

File: components/AccountVerificationForm/AccountVerificationFormStep0SignUp.js [hunk 1]
```diff
@@ -1,12 +1,12 @@
 import { useEffect, useState } from 'react';
 import { useFormState } from 'react-use-form-state';
-import { axios } from '../../utils/axios';
 import { Button } from '../Button';
 import { TextField } from '../TextField';
 import { ErrorMessage } from '../ErrorMessage';
```
Comment (k-cem-fd) [inline components/AccountVerificationForm/AccountVerificationFormStep0SignUp.js]: Can you update Button, TextField & ErrorMessage imports @ardacoskun

File: components/AccountVerificationForm/AccountVerificationFormStep1PreConsent.js [hunk 2]
```diff
@@ -1,34 +1,32 @@
 import { useState } from 'react';
-import { useTernaryState } from '../../utils/useTernaryState';
 import { Button } from '../Button';
```
Comment (k-cem-fd) [inline components/AccountVerificationForm/AccountVerificationFormStep1PreConsent.js]: Button import again @ardacoskun

File: components/AccountVerificationForm/AccountVerificationFormStep3LoadingSteps.js [hunk 3]
```diff
@@ -1,15 +1,14 @@
 import { useEffect, useState, useCallback } from 'react';
 import axios from 'axios';
 import { useDispatch, useSelector } from 'react-redux';
-import { fetchUserTransactions, userTransactionsLoading } from '../../store/actions/userTransactionsActions';
-import { useTernaryState } from '../../utils/useTernaryState';
 import { Button } from '../Button';
 import { CircularProgressBar } from '../CircularProgressBar';
```
Comment (k-cem-fd) [inline components/AccountVerificationForm/AccountVerificationFormStep3LoadingSteps.js]: Button & CircularProgressBar imports @ardacoskun

File: components/AccountVerificationForm/AccountVerificationFormStep4SelectAccount.js [hunk 4]
```diff
@@ -1,19 +1,19 @@
 import { useCallback, useEffect, useState } from 'react';
 import axios from 'axios';
 import { RadioGroup } from '@headlessui/react';
-import { formatCurrency } from '../../utils/formatCurrency';
-import { maskAccountNumber } from '../../utils/maskAccountNumber';
 import { Button } from '../Button';
 import { ErrorScene } from '../ErrorScene';
 import { ErrorMessage } from '../ErrorMessage';
```
Comment (k-cem-fd) [inline components/AccountVerificationForm/AccountVerificationFormStep4SelectAccount.js]: Button, ErrorScene and ErrorMessage imports @ardacoskun

**Review by k-cem-fd — APPROVED** [review]
Please update according to my comments and merge @ardacoskun

File: components/AccountVerificationForm/AccountVerificationFormStep0SignUp.js (hunk 1 above)
Comment (ardacoskun) [reply components/AccountVerificationForm/AccountVerificationFormStep0SignUp.js]: I updated PR based on your comments. @k-cem-fd

**Pull request merged**